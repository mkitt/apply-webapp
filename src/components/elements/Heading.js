// @flow
import glamorous from 'glamorous'
import { color, fontSize, space } from 'styled-system'
import { COLORS } from '../../constants/theme'

export default glamorous.h2(
  {
    display: 'flex',
    flexDirection: 'column',
    fontSize: 34,
    fontWeight: 200,
    color: COLORS.GREY_8,

  },
  space,
  fontSize,
  color,
)
