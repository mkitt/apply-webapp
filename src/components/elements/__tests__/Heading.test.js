import React from 'react'
import renderer from 'react-test-renderer'
import Heading from '../Heading'

describe('<Heading />', () => {
  it('shows the correct snapshot tree for the <Heading> element', () => {
    const component = <Heading>The Heading</Heading>
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('shows the correct snapshot tree for the <Heading> element with options', () => {
    const component = (
      <Heading color="magenta" fontSize={18} mt={8}>
        The Heading with options
      </Heading>
    )
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
