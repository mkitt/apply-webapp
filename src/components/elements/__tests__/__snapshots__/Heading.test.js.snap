// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> shows the correct snapshot tree for the <Heading> element 1`] = `
.css-0,
[data-css-0] {
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: column;
  font-size: 34px;
  font-weight: 200;
  color: #808080;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
  -webkit-flex-direction: column;
}

<h2
  className="css-0"
>
  The Heading
</h2>
`;

exports[`<Heading /> shows the correct snapshot tree for the <Heading> element with options 1`] = `
.css-0,
[data-css-0] {
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  flex-direction: column;
  font-size: 18px;
  font-weight: 200;
  color: magenta;
  margin-top: 512px;
  -webkit-box-orient: vertical;
  -webkit-box-direction: normal;
  -webkit-flex-direction: column;
}

<h2
  className="css-0"
>
  The Heading with options
</h2>
`;
